### PR TITLE
[10.0][ADD] purchase product weight new module

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

File: purchase_product_weight/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Sergio Corato <https://github.com/sergiocorato>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Purchase product on weight',
+    'version': '10.0.1.0.5',
+    'category': 'Purchase',
+    'license': 'LGPL-3',
+    'author': 'Sergio Corato, Odoo Community Association (OCA)',
+    'website': 'https://efatto.it',
```
Comment (Trivedi-Vacha-SerpentCS) [inline purchase_product_weight/__manifest__.py]: The website should be: https://github.com/OCA/purchase-workflow

File: purchase_product_weight/tests/test_purchase_weight.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            ]
+        })
+        self.product_on_weight = self.env['product.product'].search([
+            ('product_tmpl_id', '=', self.product_template_on_weight.id)
+        ], limit=1)
+
+    def test_purchase_order_on_weight(self):
+        purchase_order = self.env['purchase.order'].create({
+            'partner_id': self.env.ref('base.res_partner_3').id,
+            'date_order': time.strftime('%Y-%m-%d'),
```
Comment (nikul-serpentcs) [inline purchase_product_weight/tests/test_purchase_weight.py]: @sergiocorato IMO Here Use Server Datetime

File: purchase_product_weight/tests/test_purchase_weight.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            'order_line': [
+                (0, 0, {
+                    'product_id': self.product_on_weight.id,
+                    'product_qty': 20,
+                    'product_uom': self.product_on_weight.uom_po_id.id,
+                    'price_unit': self.product_on_weight.list_price,
+                    'name': self.product_on_weight.name,
+                    'date_planned': time.strftime('%Y-%m-%d'),
```
Comment (nikul-serpentcs) [inline purchase_product_weight/tests/test_purchase_weight.py]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

File: purchase_product_weight/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    'website': 'https://github.com/OCA/purchase-workflow',
+    'depends': [
+        'purchase',
+    ],
+    'data': [
+        'views/purchase.xml',
+        'views/product.xml',
+        'views/product_supplierinfo.xml',
+    ],
+    'installable': True,
```
Comment (kittiu) [inline purchase_product_weight/__manifest__.py:20]: Do you plan to be maintainer of this module, you can,
'maintainers': ['your_git_login'],

File: purchase_product_weight/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
```
Comment (kittiu) [inline purchase_product_weight/__init__.py:2]: Should you add copy right to all your .py file
```
# Copyright 2019 Sergio Corato <https://github.com/sergiocorato>
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```

File: purchase_product_weight/tests/test_purchase_weight.py (hunk 2 above)
Comment (kittiu) [reply purchase_product_weight/tests/test_purchase_weight.py]: @Nikul-Chaudhary you mean, `fields.Date.context_today(self)`?

**Comment by kittiu** [comment]
Can you squash your commit into 1 commit?

File: purchase_product_weight/__init__.py (hunk 5 above)
Comment (sergiocorato) [reply purchase_product_weight/__init__.py:2]: :+1:

File: purchase_product_weight/__manifest__.py (hunk 4 above)
Comment (sergiocorato) [reply purchase_product_weight/__manifest__.py:20]: :+1:

File: purchase_product_weight/tests/test_purchase_weight.py (hunk 2 above)
Comment (sergiocorato) [reply purchase_product_weight/tests/test_purchase_weight.py]: :+1:

File: purchase_product_weight/tests/test_purchase_weight.py (hunk 3 above)
Comment (sergiocorato) [reply purchase_product_weight/tests/test_purchase_weight.py]: :+1:

File: purchase_product_weight/__manifest__.py (hunk 1 above)
Comment (sergiocorato) [reply purchase_product_weight/__manifest__.py]: :+1:

**Comment by sergiocorato** [comment]
> Can you squash your commit into 1 commit?

:+1: thanks @kittiu

**Pull request closed without merging**